apps/backend/db_patches/0166_UpdateWhichCommentOnReviewProposalView.sql

```diff
@@ -0,0 +1,57 @@
+DO
+$$
+BEGIN
+    IF register_patch('0166_ExcludeExpiredProposalsFromReviewProposalView.sql', 'Thomas Cottee Meldrum', 'Update review column in the view', '2024-12-03') THEN
```

Comment:
Can the filename passed to register_patch be updated to match?

Reply:
Missed that thanks!